src/gtest/test_sidechain_ceased.cpp

```diff
@@ -1002,3 +1002,189 @@ TEST_F(CeasedSidechainsTestSuite, UndoEmptyCertUpdatesToCeasingScs) {
     EXPECT_TRUE(view->GetCeasingScs(initialCeasingHeight,restoredCeasingScIds));
     EXPECT_TRUE(updatedCeasingScIds.ceasingScs.count(scId) != 0);
 }
+///////////////////////////////////////////////////////////////////////////////
+//////////////////////////////// ApplyTxInUndo ////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////
+TEST_F(CeasedSidechainsTestSuite, SpendChangeOutput_CoinReconstructionFromBlockUndo)
+{
+    //Create sidechain
+    uint256 scId = uint256S("aaa");
+    int dummyHeight = 100;
```

Comment:
setting to constant values could be made via 'static const int'

Reply:
agree done